### PR TITLE
Added AVTP Pipeline Example

### PR DESCRIPTION
**Pull request by bdthomsen**
Added example Talker and Listener .ini files.
Added scripts to start and stop the AVTP Pipeline as a Talker & Listener.
README updates describing how to use the updated scripts and .ini files.

### CODE REVIEW

File: lib/avtp_pipeline/README.md [hunk 1]
```diff
@@ -63,22 +63,35 @@ The openavb_avdecc binary needs to be run in addition to the AVTP pipeline binar
...
+- `$ sudo ./stop_avtp_pipeline.sh eth1`
+    - Stop the current OpenAvnu AVTP Pipeline example.  The script will also attempt to cleanly recover if the AVTP pipeline binaries crashed.
+
+The AVTP Pipeline example is expected to be run simultaneously on two or more different Linux computers,
+with the network interfaces connected using AVB-capable switches.
+(The daemons do not currently support using two different network interfaces on the same computer, so different computers must be used.)
+You can refer to the list of [Avnu Certified Products](http://avnu.org/certified-products/) for switches with AVB/TSN support.
+
+To connect the Talker and Listener with the example implementation, you need to use an AVDECC controller.
+(These are also referred to as 1722.1 or ATDECC controllers.)
```
Comment (pinealservo) [inline lib/avtp_pipeline/README.md]: I think you meant AVDECC instead of ATDECC here?

Comment (bdthomsen) [reply lib/avtp_pipeline/README.md]: The ATDECC reference is intentional, as ATDECC was renamed from AVDECC by the IEEE P1722.1 work group.  Here is a reference:  http://grouper.ieee.org/groups/1722/1/web/index.html

Comment (pinealservo) [reply lib/avtp_pipeline/README.md]: Ah, I missed the rename. Shouldn't have doubted you. :)

Comment (andrew-elder) [reply lib/avtp_pipeline/README.md]: Maybe we should add "ATDECC was renamed from AVDECC by the IEEE P1722.1 in 2017." just for clarity. I didn't know about ATDECC either.

**Pull request merged**